packages/account/src/Components/address-details/__tests__/address-details.spec.tsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
 import AddressDetails from '../address-details';
-import { isDesktop, isMobile, PlatformContext, TLocationList } from '@deriv/shared';
-import { FormikProps, FormikValues } from 'formik';
+import { isDesktop, isMobile, PlatformContext } from '@deriv/shared';
```

Comment:
you removed isDesktop and isMobile from the component itself,
should we refactor test accordingly?

Reply:
oh I see in the code of test it ok, so just remove here :)

Reply:
Here it is required because the `DesktopWrapper` and `MobileWrapper` are still making use of them